### ISSUE TITLE
List items with no parent <ol>

### ISSUE THREAD

**Issue by roalexander** [Khan/tota11y#88]
Working with various accessibility plugins and continuously testing our site, we noticed that when we include the tota11y.js plugin, the appended area is created as a `<div>` with `<li>`s inside. This causes the various accessibility testing we're doing to fail, as it's not valid HTML.

![tota11y-li-bug](https://cloud.githubusercontent.com/assets/5713828/13506562/b8481002-e14c-11e5-98e1-c301639b40ad.png)

The plugin should probably create a valid HTML structure.

**Comment by jdan** [comment]
Hey @roalexander,

You're absolutely right, thanks for catching this. Making `.tota11y-plugins` a `<ul>` should fix this (maybe a few lines of CSS too). I'll get to this ASAP.
